### ISSUE TITLE
No hover result for `.get()` on `TypedDict` instance

### ISSUE THREAD

**Issue by debonte** [microsoft/pyright#9289]
This is a regression caused by https://github.com/microsoft/pyright/commit/8ddffa4b49ff41444acf3a8044e324109393d17b which is the fix for https://github.com/microsoft/pyright/issues/9252:

### Repro steps:
1. Hover over `get` on the last line of the code below:

```python
from typing_extensions import TypedDict

class Cls(TypedDict):
    a: int
    b: str

dct: Cls = {"a": 1, "b": "2"}
dct.get("a")
```

### Actual:
Nothing. No popup. Hovering over other tokens still works as expected.

### Expected:
This is what I see with 1.1.345 and with https://github.com/microsoft/pyright/commit/7c7703a5b962c1a9296e19d358127cf89e8a2ade which is the commit right before https://github.com/microsoft/pyright/commit/8ddffa4b49ff41444acf3a8044e324109393d17b.
![Image](https://github.com/user-attachments/assets/1ed24d6b-0ab6-47a0-99dd-09fce215cc67)

**Comment by erictraut** [comment]
Since this issue involves a language server feature, I'll leave it to you to work on @debonte. Let me know if you need help with anything in the type evaluator logic.

**Comment by debonte** [comment]
This is addressed in pyright 1.1.387.
